telegram/ext/_commandhandler.py

```diff
@@ -126,6 +137,39 @@ def __init__(
             filters if filters is not None else filters_module.UpdateType.MESSAGES
         )
 
+        self.has_args = has_args
```

Comment:
it would be good to check if `has_args` is negative and raise a `ValueError` if it is. We'd also need a corresponding unit test